src/moi_wrapper.jl

```diff
@@ -21,19 +21,34 @@ julia> x = model.add_variable(model)
 julia> model.add_constraint(model, ...)
 ```
 """
-function diff_optimizer(optimizer_constructor)::Optimizer
-    optimizer =
-        MOI.instantiate(optimizer_constructor; with_bridge_type = Float64)
+function diff_optimizer(
+    optimizer_constructor;
+    method = nothing,
```

Comment:
Do we really need this keyword argument given that the user can do `set_attribute(model, ModelConstructor(), ...)` ?

Reply:
We should probably reserve these for arguments that really need to be given at construction